doc/user_guide/customization.rst

```diff
@@ -710,18 +710,20 @@ outside the chart itself; For example, the container may be a ``<div>`` element
 
 Chart Themes
 ------------
-..
-    _comment: First mention of alt.themes
+.. note::
+
+   This material was changed considerably with the release of Altair ``5.5.0``.
```

Comment:
# Note
Trying to keep this PR focused to make for an easier review. 

I am planning to develop this section further (#3519), but would like to do that in follow-up PRs.

(All still prior to `v5.5.0`)

Reply:
Opened #3645 to track this, so marking as resolved